components/centraldashboard/webpack.config.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
                     }
                 }
             },
-            {
+            (process.platform == 'win32'?{}:{
```

Comment:
The Windows issues seem like they may be localized to your environment. I set up a Windows 2016 server on GCE, checked from master, and we able to get both the `dev` and `build` npm commands to work as expected without any changes to the eslint settings.

The site doesn't work in IE11 (per our Babel settings), but it worked perfectly fine in Chrome on Windows.

Reply:
Ok I'll work on this a little more to get it working. Seems really wierd